src/remote/util.py

```diff
@@ -117,17 +118,37 @@ def prepare_shell_command(command: Union[str, Sequence[str]]) -> str:
     return " ".join(result)
 
 
-def ssh(host: str, command: str, dry_run: bool = False, raise_on_error: bool = True):
+def ssh(
+    host: str,
+    command: str,
+    dry_run: bool = False,
+    raise_on_error: bool = True,
+    port_forwarding_config: Optional[PortForwardingConfig] = None,
```

Comment:
Importing using configuration class here is a leak of abstraction.  PortForwardingConfig should be replaced with a simple tuple here